src/rust/lib/graph-editor/src/lib.rs

```diff
@@ -1207,7 +1210,7 @@ impl application::View for GraphEditor {
         }));
 
          // === Vis Cycling ===
```

Comment:
I am just asking because it isn't clear for me. What's the meaning of visualisation cycling?

Reply:
We have different visualizations for a given type, this means that you switch to the next available visualization :)